config/sys.config.src

```diff
@@ -56,8 +56,10 @@
             devaddr_enabled => "${ROUTER_ICS_DEVADDR_ENABLED}",
             route_id => "${ROUTER_ICS_ROUTE_ID}"
         }},
-        {config_service_max_timeout_attempt, 5},
+        {config_service_max_timeout_attempt, 25},
+        {config_service_max_timeout_sleep_ms, 5000},
         {config_service_batch_sleep_ms, 500},
+        {config_service_batch_size, 500},
```

Comment:
This will send 500 SKF updates, waiting 0.5s in between, in hopes of keeping allowing the config service to not fall behind the stream. Deletes take a longer time than Additions, so we will wait for up to 2 minutes waiting for the stream to close properly.

Reply:
Can we move this under `{ics, #{`